src/internal/m365/collection/groups/collection.go

```diff
@@ -381,7 +427,7 @@ func (col *lazyFetchCollection[C, I]) streamItems(ctx context.Context, errs *fau
 					contains:      col.contains,
 					parentPath:    col.LocationPath().String(),
 				},
-				id,
+				id+metadata.DataFileSuffix,
```

Comment:
Persist as .data file for conversations only, not for channels
i.e. only add the .data suffix for conv backups. 

This is fixed in following PR in this chain. https://github.com/alcionai/corso/pull/5136/files